src/components/Person/Person.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import './Person.scss';
+import PropTypes from 'prop-types';
+
+export const Person = ({ personInfo }) => {
+  const {
+    name,
+    sex,
+    age,
+    isMarried,
+    partnerName,
+  } = personInfo;
+
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+      <p className="Person__partner">
+        {isMarried
+          ? (`My ${partner}'s name is ${partnerName}`)
+          : ('I am not married')}
+      </p>
+    </section>
+  );
+};
+
+Person.defaultProps = {
+  personInfo: {
+    age: 0,
+    partnerName: '',
+  },
```

Comment:
This will work only if the whole person won't be passed as prop. Pass not the whole object, but separate props.